src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -37,6 +37,14 @@ public class MicrobeEditor : NodeWithInput, ILoadableGameState, IGodotEarlyNodeR
     [JsonIgnore]
     private MeshInstance editorArrow;
 
+    /// <summary>
+    ///   We're taking advantage of the available membrane and organelle system already present in
+    ///   the microbe class for the cell preview.
+    /// </summary>
+    private Microbe previewMicrobe;
```

Comment:
Just to confirm, does the editor like loading a save where the previewMicrobe is visible?

Reply:
It doesn't, forgot to test it before. I'll try make it work.